File: CPP/PROGRAM/Manusia.cpp
// KELAS MANUSIA
#pragma once
#include <iostream>
#include <string>

using namespace std;

class Manusia {
    private:
        string nama;
        string noTelepon;
        string alamat;

    public:
        Manusia() {}
        
        Manusia(string nama, string noTelepon, string alamat) {
            this->nama = nama;
            this->noTelepon = noTelepon;
            this->alamat = alamat;
        }
        
        void setNama(string nama) {
            this->nama = nama;
        }
        
        void setNoTelepon(string noTelepon) {
            this->noTelepon = noTelepon;
        }
        
        void setAlamat(string alamat) {
            this->alamat = alamat;
        }
        
        string getNama() {
            return nama;
        }
        
        string getNoTelepon() {
            return noTelepon;
        }
        
        string getAlamat() {
            return alamat;
        }
        
        void tampilkanInfo() {
            cout << "Nama: " << nama << endl;
            cout << "No. Telepon: " << noTelepon << endl;
            cout << "Alamat: " << alamat << endl;
        }
        
    ~Manusia() {}
};

File: PYTHON/PROGRAM/Manusia.py
# KELAS MANUSIA
class Manusia:
    def __init__(self, nama="", no_telepon="", alamat=""):
        self.__nama = nama
        self.__no_telepon = no_telepon
        self.__alamat = alamat
    
    def set_nama(self, nama):
        self.__nama = nama
    
    def set_no_telepon(self, no_telepon):
        self.__no_telepon = no_telepon
    
    def set_alamat(self, alamat):
        self.__alamat = alamat
    
    def get_nama(self):
        return self.__nama
    
    def get_no_telepon(self):
        return self.__no_telepon
    
    def get_alamat(self):
        return self.__alamat
    
    def tampilkan_info(self):
        print("Nama:", self.__nama)
        print("No. Telepon:", self.__no_telepon)
        print("Alamat:", self.__alamat)